pkg/ingester/ingester.go

```diff
@@ -1284,6 +1356,9 @@ func (i *Ingester) AllUserStats(context.Context, *client.UserStatsRequest) (*cli
 	if err := i.checkRunning(); err != nil {
 		return nil, err
 	}
+	if err := i.checkReadOverloaded(); err != nil {
```

Comment:
This is an internal method, not exposed to the final user (just to Mimir operators). I wouldn't check it here.

Reply:
Thanks for making me aware, removing it.